### ISSUE TITLE
Review the new IMs section and add missing (useful) information

### ISSUE THREAD

**Issue by szymczdm** [JacquesCarette/Drasil#523]
When updating the IMs (on @halonazhao's branch) the section has changed quite a bit. We need to be able to re-introduce any lost information (one example is pictured here):
![reviewims](https://user-images.githubusercontent.com/235140/39978026-0dec967c-570c-11e8-8535-619f18bad7a7.PNG)
Left is build, Right is stable.

Notice there is a paragraph missing. This section now has a number of issues which may need to be looked at more closely by @JacquesCarette, @smiths, and myself before we can actually proceed in fixing it. We need to know what the missing paragraph(s) actually *mean*.

**Comment by szymczdm** [comment]
#506 and #507 mention different aspects of this issue, but it appears to be the whole section which will need review, so I'm going to leave this up until each of those are closed and we've had a chance to review the IM section once again.

**Comment by smiths** [comment]
@szymczdm, I'm not sure if your example is from noPCM or SWHS. 
 The blurb of text "The goals GS1 to .. " should be in the build version as well as the stable version.  The text is intended to orient the reader about the relationship between the different pieces of knowledge.  This text is present in the SWHS example:

 https://github.com/smiths/swhs/blob/master/docs/SRS/PCM_SRS.tex

It has been commented out of the noPCM example:

https://github.com/smiths/caseStudies/blob/c98949681bacdbf870030c774ccedcd6f67d8cd2/CaseStudies/noPCM/docs/SRS/SRS.tex#L1014-L1019

The text would have been commented out of the noPCM example because it isn't exactly right for this example.  The text should have instead been updated for the noPCM example.

**Comment by elwazana** [comment]
@szymczdm Differences Report between Stable and Manual:

- SWHS
    - Table of Units
        - Different order of displayed symbols (Trivial)
Stable:
![image](https://user-images.githubusercontent.com/35268928/41620738-a2d56f14-73d8-11e8-846f-26233a33ed88.png)
Manual:
![image](https://user-images.githubusercontent.com/35268928/41620782-bfd208a2-73d8-11e8-86bd-01ea3850af40.png)

    - Table of Symbols
        - Different order of columns (Trivial)
Stable:
![image](https://user-images.githubusercontent.com/35268928/41620849-eb5235b0-73d8-11e8-93fe-59b4aba1b7d5.png)
Manual:
![image](https://user-images.githubusercontent.com/35268928/41620873-ff5b206c-73d8-11e8-9d18-9ca828dfc5a5.png)
        - Unsure what this is but doesn't appear in the manual (non-trivial)
Stable:
![image](https://user-images.githubusercontent.com/35268928/41620925-2e837786-73d9-11e8-8d64-7a9b8223d191.png)
Manual:

        - R {Aspect ratio} missing from stable (non-trivial)
Stable: 
Manual:
![image](https://user-images.githubusercontent.com/35268928/41620979-5a9e9c10-73d9-11e8-810b-ebca7cbac6de.png)

    - IMs
        - Missing information paragraph (non-trivial)
Stable:
Manual:
![image](https://user-images.githubusercontent.com/35268928/41621082-a8619844-73d9-11e8-8cfe-1696849fbc24.png)

        - Missing information in IM1 (non-trivial)
        **1.** Assumptions associated with the input and output of IM1
Stable:
![image](https://user-images.githubusercontent.com/35268928/41621222-17f4069c-73da-11e8-9f96-550cf8c47143.png)
Manual:
![image](https://user-images.githubusercontent.com/35268928/41621276-2ef96f3a-73da-11e8-9984-597c54618e80.png)
        **2.** Description is missing associated assumptions and constraints 
Stable: 
![image](https://user-images.githubusercontent.com/35268928/41621363-68dda45a-73da-11e8-8d63-930a8c5fa166.png)
Manual:
![image](https://user-images.githubusercontent.com/35268928/41621395-81b16c8c-73da-11e8-9b2e-0ffbf8c81fa9.png)

        - Missing information about associated assumptions and DDs in the derivation (non-trivial)
Stable:
![image](https://user-images.githubusercontent.com/35268928/41621454-b05ccbbc-73da-11e8-9d8d-738f79897db0.png)
Manual:
![image](https://user-images.githubusercontent.com/35268928/41621478-c1dccc84-73da-11e8-86d2-c4407cfcd49c.png)

        - Glitch (trivial)
Stable: 
![image](https://user-images.githubusercontent.com/35268928/41621555-fb199dec-73da-11e8-83d3-130f0bf1f7c4.png)
Manual:
![image](https://user-images.githubusercontent.com/35268928/41621574-1128ea2a-73db-11e8-904d-956d70d2125f.png)

        - Missing information for the output of IM2 (non-trivial)
Stable:
![image](https://user-images.githubusercontent.com/35268928/41621687-5fabb998-73db-11e8-974e-d237e1047fea.png)
Manual:
![image](https://user-images.githubusercontent.com/35268928/41621721-728047d2-73db-11e8-8e43-60f7e0838532.png)

        - Missing information in derivation of IM2 (non-trivial)
Stable:
![image](https://user-images.githubusercontent.com/35268928/41621918-1081a8fe-73dc-11e8-8180-26b4bc602353.png)
Manual:
![image](https://user-images.githubusercontent.com/35268928/41621995-40693fa0-73dc-11e8-9ccf-f84660bf3d6d.png)

        - Additional A<sub>C</sub> in inputs of all IMs (Unsure if this is a trivial error or is the tip of a larger problem)
Stable: 
![image](https://user-images.githubusercontent.com/35268928/41622123-9eb93f24-73dc-11e8-90bf-10ee908872f2.png)
Manual:
![image](https://user-images.githubusercontent.com/35268928/41622141-aff87570-73dc-11e8-8ea9-6431bb8fadb2.png)

        - Missing input in IM3 "T<sub>W</sub> from IM1" (non-trivial)
Stable:
![image](https://user-images.githubusercontent.com/35268928/41622240-0162a0c0-73dd-11e8-83e0-a40866e86e37.png)
Manual:
![image](https://user-images.githubusercontent.com/35268928/41622279-1840d3c0-73dd-11e8-8f72-7112eaf03db4.png)

        - Missing information about assumptions associated with description of IM3 (non-trivial)
Stable:
![image](https://user-images.githubusercontent.com/35268928/41622349-4b86e92c-73dd-11e8-9e47-d911f63d5512.png)
Manual:
![image](https://user-images.githubusercontent.com/35268928/41622366-598a5432-73dd-11e8-9565-93ed16d05a9e.png)

        - Missing a lot of information in the description of IM4 including assumptions, DDs, and TMs (non-trivial)
Stable:
![image](https://user-images.githubusercontent.com/35268928/41622469-a98059aa-73dd-11e8-88a7-aebeefe189a7.png)
Manual:
![image](https://user-images.githubusercontent.com/35268928/41622510-cb545e0a-73dd-11e8-8457-ab739438f4f5.png)

**Comment by szymczdm** [comment]
Thanks for the great summary! I'll take a more in-depth look at it shortly. @smiths can you take a look as well and let me know if there's anything you'd reclassify (trivial/non-trivial) or consider a non-issue?

**Comment by smiths** [comment]
Excellent work @elwazana.  Very thorough.  The information lost between the manual and stable versions is worse than I expected.  Good to know so that now we can fix it.

I agree with your summary, I only have a few additional points to add:

- MINFRACT is in the manual version, but in lower case, and it doesn't appear in the summary table of auxiliary constants, as it should.  The table of auxiliary constants was inadvertently changed to match stable at one point, instead of the other way around.  I will correct the manual version to use ALLCAPS, since this is the usual standard for constants.  I'll also add the entry back to the Auxiliary constants table.  The changes are reflected in: https://github.com/smiths/swhs/commit/ced732a72debe870ec0cfc00228f9717f6b89a1e
This issue was discussed in https://github.com/JacquesCarette/Drasil/issues/451

- The aspect ratio (R) was discussed in https://github.com/smiths/caseStudies/issues/26 Changes were made to manual, but not propagated to Drasil.  @halonazhao might be able to help with this.  The LaTex for adding R was given in commit https://github.com/smiths/swhs/commit/f68dc8e82127798dc17575ed3ef9a61dec1636d8

- The derivation information was added for the energy balances as a consequence of our discussion about the thermal flux applying over the coil and the PCM.  The commit where this information was added is given at https://github.com/smiths/swhs/commit/2106695f9fb6c5d8f5215779bb489b698c17a7ca
This was discussed in https://github.com/smiths/swhs/issues/48

**Comment by smiths** [comment]
I think the key for this issue is to split it into separate issues.  There is too much going on here to keep track.  Unfortunately we cannot currently fix the Drasil version to match the manual version because of the referencing issues.  We need to sort out #1029 first.  In addition to the work that @halonazhao is doing, I will reclassify some of the TMs, DDs, etc.  (I'll create a separate issue for this soon.)  Once this is done, we can return to this issue.

**Comment by smiths** [comment]
I have gone through the SWHS example and identified GDs and DDs that should be reclassified.  I will create an issue for this, but there aren't actually any problems with the imports (based on #1029) that these changes will fix.  The changes will improve the conceptual integrity and consistency of the document, but they aren't necessary to proceed with this issue.  (I don't think.)

**Comment by smiths** [comment]
All of the issue created to sort out this issue have been closed.  There are too many details to go through it all again; I am going to assume that the process works and close this issue too.
